app/styles/_notifications.less

```diff
@@ -39,9 +39,9 @@ notification-drawer-wrapper {
   }
   // the whole block is clickable, need to set pointer on all of these
   // for the correct visual
```

Comment:
I'm not sure this is quite right, might be that we just want to eliminate the hover (bg blue) as well as return the cursor.